docs-ref-services/latest/communication-networktraversal-readme.md

```diff
@@ -5,10 +5,9 @@ author: JianpingChen
 ms.author: jiach
 ms.date: 02/11/2022
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: java
-ms.service: communication
+ms.service: azure-communication-services
```

Comment:
Changes to `ms.service` should also go into https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/java-packages.csv to prevent being overwritten with incorrect values on publishing. Make changes to the metadata CSV files first and then we can approve these.